### ISSUE TITLE
Query regarding test and validation split

### ISSUE THREAD

**Issue by ananyajana** [Luchixiang/PCRL#8]
Hi,

Is the test/validation set same for your experiments?
It seems that config file mentions 7,8,9 for both test and validation split.

Thanks,
Ananya

**Comment by Luchixiang** [comment]
Sorry for my late reply. In the finetuning stage, we used 7 as the validation set and 8, 9 as the test set.

Best,
Jasper
